spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,21 @@
+require 'beaker-rspec/spec_helper'
+require 'beaker-rspec/helpers/serverspec'
+require 'beaker/puppet_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+
+RSpec.configure do |c|
+  # Project root
+  module_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+
+  # Configure all nodes in nodeset
+  c.before :suite do
+    # Install module and deps
+    hosts.each do |host|
+      install_dev_puppet_module_on(host, source: module_root, module_name: 'autofs',
```

Comment:
I try to always use:
install_module_on(hosts)
install_module_dependencies_on(hosts)
which installs the modules + all dependencies from metadata.json